pet-store-vanilla-js/app.js

```diff
@@ -0,0 +1,128 @@
+import {
+  configureNewPetModal,
+  configureEditPetModal,
+  showDeleteModal,
+} from './modals.js';
+import { getPetKinds, getAllPets } from './api.js';
+import { formatDate, hideError, showError } from './utils.js';
+import { formElements } from './form.js';
+
+const mainPageElements = {
+  addPetButton: document.getElementById('add-pet-btn'),
+  loadingPetsSpinner: document.getElementById('loading-pets-spinner'),
+  tableBody: document.getElementById('data-table').tBodies[0],
+};
+
+export const petKindsEnum = {};
```

Comment:
Can you use `Map`s for these instead of objects and arrays? The array thing is very bad in particular: if you create an array and assign it's 100000th element to some value (in this case 100000 is the petID) it will resize the array to have 100000 elements.